packages/theme/src/theme/_forms.scss

```diff
@@ -333,50 +346,55 @@ form {
 }
 
 .switch {
-	input[type='checkbox'] {
-		position: absolute !important;
+	input[type='checkbox'],
+	.checkbox input[type='checkbox'],
+	&.checkbox input[type='checkbox'] {
+		position: absolute;
 		clip: rect(1px, 1px, 1px, 1px);
-		padding: 0 !important;
-		border: none !important;
-		height: 1px !important;
-		width: 1px !important;
+		padding: 0;
+		border: none;
+		height: 1px;
+		width: 1px;
 		overflow: hidden;
 
 		+ * {
-			position: relative;
-			padding-left: 15px;
+			padding-left: 20px;
 		}
 
 		+ *:before,
 		+ *:after {
-			display: inline-block;
 			position: absolute;
 			content: '';
-			top: 0;
-			left: -25px;
-			transition: background 0.2s ease, transform 0.2s ease;
+			top: 0 !important;
```

Comment:
`!important` because classic checkbox will change `top` value when is checked, I don't want to override it twice